src/main/java/no/entur/uttu/export/blob/S3BlobStoreRepositoryConfig.java

```diff
@@ -70,11 +70,19 @@ public AwsCredentialsProvider cloudCredentials() {
   }
 
   @Bean
-  public S3AsyncClient s3AsyncClient(AwsCredentialsProvider credentialsProvider) {
-    S3CrtAsyncClientBuilder b = S3AsyncClient
-      .crtBuilder()
+  public S3Client s3Client(AwsCredentialsProvider credentialsProvider) {
+    S3ClientBuilder b = S3Client
```

Comment:
Please refrain from using single-letter or abbreviated variable names unless usage is idiomatic (e.g. indexes in for loops).